### DISCUSSION TITLE
Create a “static” space that is linked to a github tag

### DISCUSSION THREAD

**Discussion by rachellawson** [GitbookIO/community#163] [Feature Requests]
### What's your feature/idea?

As we publish versions of our documentation, it would be **very** useful to be able to make releases based upon the commonly used git tags, v1.0 etc. 

A space connected to a tag should always be “read only” in that the content cannot be altered from the GitBook UI. This ensures that versioned content is, indeed, versioned.

### What's your desired outcome?

The ability to create a space and base it not on a repo and branch but on a repo and tag. The UI should warn the user that this will mean the content of the space cannot be edited via the GitBook UI.

### What's the impact of this for you?

- [ ] Critical
- [ ] High
- [X] Medium
- [ ] Low

**Comment by StarpTech** [comment]
Seems to be related to https://github.com/orgs/GitbookIO/discussions/209 Adding support for versioning would be so useful, especially for technical docs.
